app/volume.go

```diff
@@ -238,7 +238,8 @@ func (job *Job) backupAndCleanup() error {
 	if err := job.snapshotAndCleanup(); err != nil {
 		return err
 	}
-	if err := job.engine.SnapshotBackup(job.snapshotName, job.backupTarget, job.labels); err != nil {
+
```

Comment:
Better to add a comment here noted that the cronjob template covered the credential already.